.github/workflows/dapr-bot/main.go

```diff
@@ -2,12 +2,15 @@ package main
 
```

Comment:
Notice that it is the bot, it should not depend on kit.

Reply:
oh yeah , thanks for that